helm/chaos-mesh/values.yaml

```diff
@@ -24,6 +24,9 @@ controllerManager:
   service:
     type: ClusterIP
 
+  # enableProfiling is a flag to enable pprof in controller-manager
+  enableProfiling: false
```

Comment:
I tend to make `enableProfiling` as a global switch to control to pprof both for `manager` and daemon`.

Reply:
Thanks for your review! Updated.